src/main/java/ai/promoted/java/client/TwoArmExperiment.java

```diff
@@ -0,0 +1,142 @@
+package ai.promoted.java.client;
+
+import ai.promoted.delivery.model.CohortArm;
+import ai.promoted.delivery.model.CohortMembership;
+
+/**
+ * Represents a two arm Experiment configuration.
+ *
+ * WARNING - while ramping up an experiment, do not change
+ * numControlBuckets or numTreatmentBuckets.  This will
+ * likely produce bad results.
+ *
+ * For the buckets, the treatment buckets are after the
+ * control buckets.
+ */
+public class TwoArmExperiment {
+  /**
+   * Name of cohort.
+   */
+  private final String cohortId;
+  
+  private final int cohortIdHash;
+  
+  /**
+   * Number of the numControlBuckets that are active.
+   */
+  private final int numActiveControlBuckets;
+  
+  /**
+   * Number of control buckets.
+   */
+  private final int numControlBuckets;
+  
+  /**
+   * Number of the numTreatmentBuckets that are active.
+   */
+  private final int numActiveTreatmentBuckets;
+  
+  /**
+   * Number of treatment buckets.
+   */
+  private final int numTreatmentBuckets;
+  
+  private final int numTotalBuckets;
+
+  /**
+   * Factory method for a 50/50 experiment.
+   * @return the configured experiment
+   */
+  public static TwoArmExperiment create5050TwoArmExperimentConfig(String cohortId, int controlPercent,
+      int treatmentPercent) {
+    if (controlPercent < 0 || controlPercent > 100) {
+      throw new IllegalArgumentException("Control percent must be in the range [0, 100]");
+    }
+    if (treatmentPercent < 0 || treatmentPercent > 100) {
+      throw new IllegalArgumentException("Treatment percent must be in the range [0, 100]");
+    }
+    return new TwoArmExperiment(cohortId, controlPercent, 100, treatmentPercent, 100);
```

Comment:
Interesting.  I think it's easier to think about there being 100 total buckets than 200 total buckets.  I was expecting 100 buckets and for the controlPercent and treatmentPercent to both be at 50 when fully launched.

Reply:
If you want to keep this, then I'd change the param to be an activation rate.

Reply:
Ok, I see what you're getting at...but I think in that case this comment from the TypeScript SDK: "Percents are specified using [0,100]" is maybe wrong, wouldn't it be [0,50] in that case?

Reply:
Ah, I meant that the units were [0,100] not [0,1.0].  That's a good point.  I'll change it to [0,50]